checker.cs

```diff
@@ -3,17 +3,70 @@
 
 class Checker
 {
-    static bool vitalsAreOk(float bpm, float spo2, float respRate) {
-        if(bpm < 70 || bpm > 150) {
-            return false;
-        } else if(spo2 < 90) {
-            return false;
-        } else if(respRate < 30 || respRate > 95) {
-            return false;
+    static int checkvitals=1;
+    static void checkbpm(float bpm)
+    {
```

Comment:
The check... functions are very similar. Can you abstract the commonality into a 'checkRange' in a class?

Reply:
Use of different functions to produce different conditions of high and low for different vitals.